src/server/game/DataStores/DB2Structure.h

```diff
@@ -2042,6 +2061,25 @@ struct RandPropPointsEntry
     uint32 UncommonPropertiesPoints[5];
 };
 
+struct RewardPackEntry
+{
+    uint32 ID;
+    uint32 Money;
+    float ArtifactPowerMultiplier;
+    uint8 ArtifactQuestXPIndex;
+    uint8 ArtifactCategoryID;
+    uint32 TitleID;
+    uint32 Unk;
```

Comment:
This field is not used in the client, checked as far as the first legion beta build